main.go

```diff
@@ -25,11 +25,13 @@ var conf = config.Get()
 func main() {
 	logging.InitLogger(conf)
 
-	if conf.KafkaSaslEnabled {
+	if conf.KafkaSaslSettings != nil {
+		saslSettings := conf.KafkaSaslSettings
+
 		logging.Log.Infof(`Sasl settings detected, using managed Kafka`)
-		logging.Log.Infof(`Kafka Sasl certificate path: %s`, conf.KafkaSaslCaPath)
-		logging.Log.Infof(`Kafka Sasl mechanism: %s`, conf.KafkaSaslMechanism)
-		logging.Log.Infof(`Kafka Sasl security protocol: %s`, conf.KafkaSaslSecurityProtocol)
+		logging.Log.Infof(`Kafka Sasl certificate path: %s`, saslSettings.CaPath)
+		logging.Log.Infof(`Kafka Sasl mechanism: %s`, saslSettings.Mechanism)
+		logging.Log.Infof(`Kafka Sasl security protocol: %s`, saslSettings.SecurityProtocol)
 	}
```

Comment:
Can we move this to the config package during the parsing? I'm not sure it belongs in main.

Reply:
Ah I get it - maybe we should add a `config.Log()` call that we can call at the end of initialization.

Reply:
Great idea — I've created a story for it: https://issues.redhat.com/browse/RHCLOUD-20347 . However I feel like this is out of the scope of this PR, so I'm just going to remove those logs from `main`.